server/src/main/java/com/jinyframework/keva/server/command/Info.java

```diff
@@ -1,21 +1,33 @@
 package com.jinyframework.keva.server.command;
 
-import lombok.val;
+import com.jinyframework.keva.server.replication.master.Replica;
 
 import java.lang.management.ManagementFactory;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentMap;
 
 import static com.jinyframework.keva.server.ServiceInstance.getConnectionService;
+import static com.jinyframework.keva.server.ServiceInstance.getReplicationService;
 
 public class Info implements CommandHandler {
     @Override
-    public Object handle(List<String> args) {
-        val stats = new HashMap<String, Object>();
-        val currentConnectedClients = getConnectionService().getCurrentConnectedClients();
-        val threads = ManagementFactory.getThreadMXBean().getThreadCount();
+    public String handle(List<String> args) {
+        final HashMap<String, Object> stats = new HashMap<>();
+        final long currentConnectedClients = getConnectionService().getCurrentConnectedClients();
+        final int threads = ManagementFactory.getThreadMXBean().getThreadCount();
         stats.put("clients:", currentConnectedClients);
         stats.put("threads:", threads);
-        return stats;
+        final ConcurrentMap<String, Replica> replicas = getReplicationService().getReplicas();
+
+        stats.put("replicas:", replicas.size());
+        int count = 0;
+        for (Map.Entry<String, Replica> entry : replicas.entrySet()) {
```

Comment:
Ye true, I'll look into it

Reply:
It's only for somewhat 'snapshot' info so I think it's fine here, same goes for entrySet, it will not change during iteration even if the hashmap has been updated by another thread.